doc/deploy/android.md

```diff
@@ -1,33 +1,128 @@
-## Deploying to the Google Play Store
-##### Prerequirements
-- You **must** read the [general deploy info](README.md).
-- You **must** have the official PVTrAck signing key. See [the keys doc](android-keys.md) for more info.
-- You **must** have the Android SDK and Android build tools installed (both bundled with Android Studio). We target API level 25 (Nougat, 7.1).
-- You **must** have access to the [Google Play Developer Console](play.google.com/apps/publish) (see @sherson).
-- You *should* have [Android Studio](https://developer.android.com/studio/index.html) installed.
+# Building an Android App to Deploy in Google Play
 
-##### Steps
+## Requirements
+_These instructions assume you are running a Mac. If you're not, the installation
+instructions will be vastly different. You can ask for help or, since you're probably
+running Linux, just translate them like you're probably used to doing._
 
-1. `npm install`
+### General
+- Did you read the [general deploy info](README.md)?
+- Do you have access to the [Google Play Developer Console](play.google.com/apps/publish)?
+ If not, see Adam.
 
-2. Open `config.xml`. On line 2, increment `android-versionCode` with your appropriately chosen new version number.
+### Android Studio
+We use Android Studio to easily manage the Android SDK tools due to Android Studio
+automatically checking for updates and allowing easy management. The tools can
+be downloaded separately, but these instructions do not cover how.
+1. Get [Android Studio](https://developer.android.com/studio/)
+1. Use it to download the SDK platforms and tools: 
+  1. Navigate through these menu items:
```

Comment:
Are these indentations four spaces long?

Reply:
No. You can actually tell from here if you highlight with your mouse.